frontend/src/views/Blacksmith.vue

```diff
@@ -372,11 +372,11 @@
                   </div>
                 </div>
                 <div>
-                  <input v-model="lesserDust" type="range" min="0" :max="getLesserDust()" value="0" steps="1">
+                  <input v-model="lesserDust" type="range" min="0" :max="100" value="0" steps="1">
```

Comment:
Imo it would be nice to extract these "magic numbers" to some constants, even in this file, just for the sake of clean code

Reply:
Good point, I changed it, hopefully this is what you meant!